backend/app/models/content.py

```diff
@@ -33,16 +51,25 @@ def to_dict(self):
     def __repr__(self):
         """Method to return a string representation of the model"""
         return f"\
-            Item('{self.name}', '{self.description}'), '{self.created_at}',\
+            Item('{self.title}', '{self.body}'), '{self.created_at}',\
                   '{self.updated_at}', '{self.deleted_at}'"
 
 
+# pylint: disable=wrong-import-position
+# pylint: disable=unused-import
```

Comment:
Why are you disabling unused import an cyclic import check?

- [ ] use the checks as they help with code quality, sometimes I know as  you gotta disable some strict pylint rules, but I think these rules aren't strict but necessary.

Reply:
I have changed the import position to be at the top of the file and enabled the wrong-import-position rule. I have also changed the inverse relationship implementation to be implicit using the backref property of sqlalchemy relationships to eliminate the need for cyclic imports. But the unused-import rule is still disabled since the linter does not recognize that the import is needed by the relationship definition where I've used the class name in string format. Sqlalchemy looks for the class inside of the module. So it was necessary.